contentcuration/contentcuration/frontend/shared/views/contentNodeFields/LevelsOptions.vue

```diff
@@ -1,41 +1,53 @@
 <template>
 
-  <DropdownWrapper>
-    <template #default="{ attach, menuProps }">
-      <VSelect
-        ref="level"
-        v-model="level"
-        :items="levels"
-        box
-        chips
-        clearable
-        :label="translateMetadataString('level')"
-        multiple
-        deletableChips
-        :menu-props="menuProps"
-        :attach="attach"
-      />
-    </template>
-  </DropdownWrapper>
+  <div>
+    <slot name="prependOptions"></slot>
+    <ExpandableSelect
+      v-model="level"
+      multiple
+      :expanded="expanded"
+      :options="levels"
+      :hideLabel="hideLabel"
+      :label="translateMetadataString('level')"
+    />
+  </div>
 
 </template>
 
 <script>
 
   import camelCase from 'lodash/camelCase';
   import { ContentLevels } from 'shared/constants';
+  import ExpandableSelect from 'shared/views/form/ExpandableSelect';
   import { constantsTranslationMixin, metadataTranslationMixin } from 'shared/mixins';
-  import DropdownWrapper from 'shared/views/form/DropdownWrapper';
 
   export default {
     name: 'LevelsOptions',
-    components: { DropdownWrapper },
+    components: { ExpandableSelect },
     mixins: [constantsTranslationMixin, metadataTranslationMixin],
     props: {
+      /**
+       * It can receive a value as an array of strings of the selected levels, or
+       * an object with the following structure:
+       * {
+       *  [levelId]: true | [nodeId1, nodeId2, ...]
```

Comment:
Same as above - my thought here is that having to deal with different input types feels like it adds complexity in handling.